hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/spi/impl/ReconDBProvider.java

```diff
@@ -83,11 +83,16 @@ static void truncateTable(Table table) throws IOException {
     if (table == null) {
       return;
     }
-    try (TableIterator<Object, ? extends KeyValue<Object, Object>>
-            tableIterator = table.iterator()) {
-      while (tableIterator.hasNext()) {
-        KeyValue<Object, Object> entry = tableIterator.next();
-        table.delete(entry.getKey());
+    try (TableIterator<Object, ? extends KeyValue<Object, Object>> iterator = table.iterator()) {
+      if (iterator.hasNext()) {
+        Object firstKey = iterator.next().getKey();
+        Object lastKey = null;
+        while (iterator.hasNext()) {
+          lastKey = iterator.next().getKey();
+        }
+        if (lastKey != null) {
+          table.deleteRange(firstKey, lastKey);  // Efficient bulk deletion
```

Comment:
We should consider implementing `dropColumnFamily` a more performative API call on rocks DB column families compared to `deleteRange` when we want to truncate the whole table data.

Reply:
Will take care of it in a separate PR for now, after digging into the code a little bit.